.github/workflows/integration_tests.yml

```diff
@@ -99,18 +77,22 @@ jobs:
       KEPTN_SERVICE_TYPE: ${{ matrix.KEPTN_SERVICE_TYPE }}
       RUN_CONTINUOUS_DELIVERY_TEST: ${{ matrix.RUN_CONTINUOUS_DELIVERY_TEST }}
       RUN_AIRGAPPED_TEST: ${{ matrix.RUN_AIRGAPPED_TEST }}
+      AIRGAPPED_REGISTRY_URL: "k3d-container-registry.localhost:12345"
       REMOTE_EXECUTION_PLANE: ${{ matrix.REMOTE_EXECUTION_PLANE }}
-      BACKUP_RESTORE: ${{ matrix.BACKUP_RESTORE }}
-      RESOURCE_SERVICE: ${{ matrix.RESOURCE_SERVICE }}
-      RUN_QUALITY_GATES_TEST: ${{ matrix.RUN_QUALITY_GATES_TEST }}
-      KEPTN_EXAMPLES_BRANCH: ${{ github.event.inputs.examples_branch }}
       COLLECT_RESOURCE_LIMITS: ${{ matrix.COLLECT_RESOURCE_LIMITS }}
       GO_VERSION: ^1.16
+      TEST_REPORT_FOLDER: test-reports-${{ matrix.CLOUD_PROVIDER}}-${{ matrix.PLATFORM_VERSION }}
+      FINAL_PLATFORM_TEST_REPORT_FOLDER: test-reports
```

Comment:
why this was has `PLATFORM` in its name?

Reply:
I named it this way because it will contain test reports with this file format: `test-reports/test-report-final-GKE-1.12.log`.
But it will actually also contain the final test report in the end so the name is a little misleading, I will change it to `FINAL_TEST_REPORT_FOLDER`